README.md

```diff
@@ -54,9 +54,9 @@ We welcome your feedback and contributions in any form possible.
   - Already signed up? Head to our discussions at [#openebs-users](https://openebs-community.slack.com/messages/openebs-users/)
 - Want to raise an issue?
   - If it is a generic (or `not really sure`), you can still raise it at [issues](https://github.com/openebs/openebs/issues)
-  - Project specific issues can be raised at individual project level.
+  - Project (repo) specific issues also can be raised at [issues](https://github.com/openebs/openebs/issues) and tagged with the individual repo label like *repo/maya*.
 - Want to help with fixes and features?
-  - Have a look at [open issues](https://github.com/issues?q=user%3Aopenebs+is%3Aopen)
+  - Have a look at [open issues](https://github.com/openebs/openebs/labels)
```

Comment:
url "https://github.com/openebs/openebs/issues" ?

Reply:
showing the labels may be better.. to have the contributor easily pick the issue they want to work on..

Reply:
cool !!!